File: c-sharp/Api/Movement.cs
using CsvHelper.Configuration.Attributes;

namespace Npb.Agview.Api.Example
{
    public class Movement
    {
        [Name("movement_id")]  
        public int MovementId { get; set; }

        [Name("species")]
        public string Species { get; set; }

        [Name("number_in_shipment")]
        public int NumberInShipment { get; set; }

        [Name("movement_datetime")]
        public string MovementDatetime { get; set; }

        [Name("movement_type")]
        public string MovementType { get; set; }
    }
}

File: java/src/main/java/org/npb/agview/api/example/Movement.java
package org.npb.agview.api.example;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class Movement {
    private int movementId;
    private String species;
    private int numberInShipment;
    private String movementDatetime;
    private String movementType;
}
